internal/support/notifications/operators/notification/get_test.go

```diff
@@ -267,6 +267,55 @@ func TestStartExecutor(t *testing.T) {
 	}
 }
 
+func TestEndExecutor(t *testing.T) {
+	tests := []struct {
+		name           string
+		mockDb         NotificationLoader
+		expectedResult []contract.Notification
+		expectedError  bool
+	}{
+		{
+			name:           "Successful database call",
+			mockDb:         createMockNotificiationLoaderStartStringArg("GetNotificationsByEnd", nil, SuccessfulDatabaseResult, End, Limit),
+			expectedResult: SuccessfulDatabaseResult,
+			expectedError:  false,
+		},
+		{
+			name:           "Unsuccessful database call",
+			mockDb:         createMockNotificiationLoaderStartStringArg("GetNotificationsByEnd", Error, []contract.Notification{}, End, Limit),
+			expectedResult: []contract.Notification{},
+			expectedError:  true,
+		},
+		{
+			name:           "Unsuccessful database call",
```

Comment:
Not really unsuccessful DB call, more like successful but the item wasn't found.